ansible/roles/debops.prosody/defaults/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+
```

Comment:
The `defaults/main.yml` files should have a header that enables vim folding, for usability. There should also be a header that creates a separate section in the autogenerated documentation. See other roles for examples.

Reply:
k thanks